elements/lisk-p2p/README.md

```diff
@@ -1,13 +1,106 @@
 # @liskhq/lisk-p2p
 
-@liskhq/lisk-p2p is containing unstructured P2P library for use with Lisk-related software
+@liskhq/lisk-p2p is containing unstructured P2P library for creating and running unstructured P2P networks. The aim of this library is to make it easier for developers to create P2P projects without thinking about security, discovery, selection and eviction mechanisms and concentrate more on developing use cases and P2P projects on top of it. A developer can leverage underlying robust security, discovery and selections mechanisms and use the easy to use interface to use the library. This library implements features proposed in [LIP: 0004: Introduce robust peer selection and banning mechanism](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0004.md).
 
 ## Installation
 
 ```sh
 $ npm install --save @liskhq/lisk-p2p
 ```
 
+To run a P2P node.
+
+```typescript
+const { P2P } = require('@liskhq/lisk-p2p');
+
+const p2p = new P2P({
+	nodeInfo: {
+		wsPort: 5001,
+		nethash: '123xyz', // network identifier
+		protocolVersion: '1.1',
+	},
+});
+
+const run = async () => {
+	await p2p.start();
+};
+
+run()
+	.then(() => console.log('P2P node has started!'))
+	.catch(err => {
+		console.log(`Error occurred while running p2p node: ${err}`);
+		p2p.stop();
+		process.exit(1);
+	});
+```
+
+It will start a node that is available on `5001` websocket port.
+
+```shell
+P2P node has started!
+```
+
+### Actions
+
+It provides simple interface to send, request, broadcast information and many more functions to interact with the network.
+
+- `p2p.start()` to start a P2P node after creating an instance.
+- `p2p.stop()` to stop a P2P node.
+- `p2p.config` to get the config of the node.
+- `p2p.isActive` to check the status if the node is up and running.
+- `p2p.nodeInfo` to check the node status and information.
+- `applyNodeInfo(nodeInfo: P2PNodeInfo)` to broadcast your updated `nodeInfo` to the network.
+- `p2p.getConnectedPeers()` to get all the connected peers that are connected to your node in the network.
+- `p2p.getDisconnectedPeers` to get all the disconnected peers that are part of the network but not connected to you.
+- `p2p.request(packet: P2PRequestPacket)` to request information from the network that will run the peer selection and finds an appropriate peer for you to request information.
+- `p2p.send(message: P2PMessagePacket)` it will send the information to 16 connected peers choosen by peer selection for send.
+- `broadcast(message: P2PMessagePacket)` to broadcast information to all the connected peers.
```

Comment:
format seems to be inconsistent. Prefer to make it either `p2p.xxx` or just a function name